crates/medmodels-core/src/medrecord/graph/mod.rs

```diff
@@ -313,18 +316,34 @@ impl Graph {
 
     pub fn edges_connecting<'a>(
         &'a self,
-        source_node_index: Vec<&'a NodeIndex>,
+        source_node_indices: Vec<&'a NodeIndex>,
         target_node_index: Vec<&'a NodeIndex>,
```

Comment:
Why change source_node_index naming and not also target_node_index?

Reply:
good catch, thank you :)
Fixed